magda-web-client/src/api-clients/OrgUnitApis.ts

```diff
@@ -27,3 +27,35 @@ export async function listOrgUnits({
         return await res.json();
     }
 }
+
+export interface OrgUnitWithRelationship extends OrgUnit {
+    relationship?: string;
+}
+
+/**
+ * List all OrgUnits at certain org tree level.
+ * Optionally provide a test Org Unit Id that will be used to
+ * test the relationship with each of returned orgUnit item.
+ * Possible Value: 'ancestor', 'descendant', 'equal', 'unrelated'.
+ *
+ * @export
+ * @param {number} orgLevel - The level number (starts from 1) where org Units of the tree are taken horizontally.
+ * @param {string} [testOrgUnitId] - Optional; The org unit id that is used to test the relationship with each of returned orgUnit item.
+ * @returns {Promise<OrgUnitWithRelationship[]>}
+ */
+export async function listOrgUnitsAtLevel(
+    orgLevel: number,
+    testOrgUnitId?: string
```

Comment:
Same thing as above re calling it `testOrgUnitId` :)

Reply:
Will update accordingly 👍